### DISCUSSION TITLE
Support for Flashforge Dreamer?

### DISCUSSION THREAD

**Discussion by CCoenC** [supermerill/SuperSlicer#1846] [Q&A]
I have a flashforge dreamer and currently use Simplified 3d but it looks as if that will never be updated again. Is there a way to make super slicer work with it? I know there is no option currently for flashforge firmware however is there a way to make certain processes output custom codes. For example, since Makerbot is somewhat similar, could you just use that and then set certain actions such as setting temp and waiting for it to reach temp output a custom code which would be for the flashforge firmware?

**Comment by supermerill** [comment]
There is a plan to change the way the gcode is generated.
This will allow use to create scripts that may output something else than gcode.
But it won't be ready for Christmas at the current pace.

**Reply by CCoenC** [reply]
Alright cool, glad to know it's coming. 
Keep up the great work!

**Reply by jlweatherby99** [reply]
I am also wanting this feature for my flashforge creator pro2! Thank you for continuing the great work!

**Reply by NickintheNorth** [reply]
I'd love to switch to Superslicer as Flashprint feels quite limited compared to what I read here and there about industry standard slicers. Unfortunately I havent found any which would be compatible apart from simplify3d which has the limitation OP presented. Is it currently usable with the Dreamer and its dual extruder? What should we look for to know when it will be compatible (while reading the patch notes)?
Thanks for your efforts!
